docs/secure-connections/sauce-connect/setup-configuration/specialized-environments.md

```diff
@@ -15,6 +15,63 @@ import TabItem from '@theme/TabItem';
 * The name of your closest regional Sauce Labs Data Center (see the [SC CLI](/dev/cli/sauce-connect-proxy/#--region) and [Data Center Endpoints](/basics/data-center-endpoints/).
 
 
+## Sauce Connect Docker Container Setup
+
+As an alternative to downloading and installing the SC Client, you can leverage Docker containers to manage Sauce Connect Proxy tunnels. See [Using Docker Containers to Install/Run Sauce Connect](/secure-connections/sauce-connect/installation/#using-docker-containers-to-installrun-sauce-connect) for use cases.
+
+### Running the SC Docker Image
+
+1. Before running the container, you'll need to pull the Sauce Connect Proxy Docker Image from the Docker Hub. This will always pull the latest version of Sauce Connect Proxy.
+  ```bash
+  $ docker pull saucelabs/sauce-connect
+  ```
+   * Or - if you _do_ want to use a specific SC version - you can specify that as a tag. The example below pulls v4.7.0.
+   ```bash
+   $ docker pull saucelabs/sauce-connect:4.7.0
+   ```
+2. To run the Sauce Connect Proxy Docker image, execute the below script, which will also set your Sauce Labs username and access key as [environment variables](/basics/environment-variables/).
+  ```bash
+  $ export SAUCE_USERNAME="my-user"
+  $ export SAUCE_ACCESS_KEY="my-access-key"
+  docker run \
+      -e SAUCE_USERNAME=${SAUCE_USERNAME} \
+      -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
+      --network="host" \
+      -it saucelabs/sauce-connect
+  ```
+
+If desired, you can specify any additional [SC CLI arguments](/dev/cli/sauce-connect-proxy/) here. Ensure that you have `--network="host"` set as an argument, otherwise Sauce Connect within the Docker container will not be able to access your local services in the host machine.
```

Comment:
I think `--network=host` would only be necessary if your tests are using `localhost` addresses. Using that option may need special permission so by default I wouldn't use it.

So I think we should specifically call out that option to only use it if `localhost` endpoints are needed.